src/containers/NavigateBackButton.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Button to go back in router history programmatically
+ */
+import React from 'react';
+import IconButton from 'material-ui/IconButton';
+import ArrowBack from 'material-ui-icons/ArrowBack';
+import { withRouter, Redirect } from 'react-router-dom';
+
+// TODO: Check for history, if false redirect to /assets/dept
```

Comment:
Looking at the [documentation](https://github.com/ReactTraining/history) for history, it doesn't look like it is trivial to determine whether goBack() will leave the app. I'd be tempted to leave this TODO for the moment.